pydoctor/templates/apidocs.css

```diff
@@ -195,30 +202,90 @@ tr.private {
     margin-bottom: 6px;
 }
 
+/* This is magic to show the transparent index link */
```

Comment:
I think it's useful to have a link to the index somewhere, but this is only half a solution, so I'd prefer to solve it completely in a separate PR.